src/components/FormType.tsx

```diff
@@ -38,11 +40,17 @@ const CalendarFormType = () => (
 const GitHubFormType = ({ modals }: GitHubFormTypeProps) => {
 
   const { setCreateModalState, setRulesetModalState } = modals
+  const { data: session } = useSession()
+  
+  // show some ui error handling
+  const result = api.webhook.createWebhook.useMutation()
   
   const handleSubmit = (e: React.FormEvent) => {
+    if (!session) return "log out and log back in"
```

Comment:
what does this returned value do? is it displayed?

also, a nit: I dislike one-liner if statements because it's harder for readers to understand the control-flow at a glance

Reply:
the return value is a todo. It isn't displayed but I will probably create a small component telling the user to sign out and sign back in. The if statement is just a placeholder. In the end, it would probably be a useState hook to check for the session and if it is undefined, then show some ui handling.

Reply:
if the code is a placeholder/TODO best to mark it as such and leave a comment about your intentions, that makes it easier for reviewers